index.js

```diff
@@ -197,27 +198,19 @@ function parse_args(process_args) {
     .option('-k, --statstoken <STATS_TOKEN>', 'Specify log token for forwarding statistics', process.env.INSIGHT_STATSTOKEN)
     .option('-t, --token <TOKEN>', 'Specify token to use', process.env.INSIGHT_TOKEN)
     .option('-v, --log-level <LEVEL>', 'Define application log level', process.env.INSIGHT_LOG_LEVEL || 'info')
-    //  TODO (sbialkowski): Remove in next release
-    .option('--debug', 'DEPRECATED: Set application log level to "debug" (use `--log-level debug`)', false)
     .option('--matchByName <REGEX>', 'Forward logs for containers whose name matches <REGEX>')
     .option('--matchByImage <REGEX>', 'Forward logs for containers whose image matches <REGEX>')
     .option('--skipByName <REGEX>', 'Do not forward logs for containers whose name matches <REGEX>')
     .option('--skipByImage <REGEX>', 'Do not forward logs for containers whose image matches <REGEX>')
-    .option('--no-docker-events, --no-dockerEvents', 'Do not stream Docker events')
```

Comment:
i feel like there's no actual benefit to removing this option but you're going to make support's life unnecessarily difficult by doing it... it's not like all options are already consistent right, some are dash-separated and some are camel-cased still. just keep both tbh 🤷

Reply:
yeah probably small advantage, more consistent naming

i just think if i'm bumping major anyway why not bring the naming a bit more consistent

i'll write up a migration guide for the release anyway, it's not a big change